src/CREATE3Factory.sol

```diff
@@ -9,25 +9,25 @@ import {Ownable} from "openzeppelin-contracts/contracts/access/Ownable.sol";
 
 /// @title Factory for deploying contracts to deterministic addresses via CREATE3
 /// @author zefram.eth
```

Comment:
Should remove?

Reply:
Should we keep the `@author zefram.eth` because of the AGPL-3.0 License?

Reply:
I think we do but i'm not sure if this is the right way to do that. If so then i have no additional questions.